### DISCUSSION TITLE
Holidays for the whole class

### DISCUSSION THREAD

**Discussion by SelectLOL1** [flowhorn/schulplaner#187] [Q&A] [answered]
_Hello, first of all, thanks for this amazing open-source project._ 

My questions is: How can I add holidays for the whole schoolclass? Holidays that I add seem to be visible only for me.

Thank you very much and greetings
SelectLOL

**Comment by Th3Ph4nt0m** [comment, marked as answer]
Hey @SelectLOL1,

I don't think this is possible for now. Thank you for creating #188! We have it on the Agenda and will add this feature as soon as possible for us. Unfortunately, Felix (flowhorn) and I do not have that much time besides school and therefore don't know how long it will take. If you know someone who loves coding apps with Flutter, feel free to tell them about this project! Maybe others are faster than us 😉.

Greetings!
Henrik
